src/app/portal/components/case/case-file/case-transcripts-table/case-transcripts-table.component.html

```diff
@@ -1,25 +1,45 @@
-<app-govuk-heading tag="h2" size="m">All transcripts for this case</app-govuk-heading>
+@if (!adminScreen()) {
+  <app-govuk-heading tag="h2" size="m">All transcripts for this case</app-govuk-heading>
+}
 
 <app-data-table
   id="transcriptsTable"
   [rows]="transcripts()"
-  [columns]="columns"
+  [columns]="adminScreen() ? adminColumns : columns"
   caption="Transcripts associated with case"
   [hiddenCaption]="true"
   noDataMessage="There are no transcripts for this case."
 >
   <ng-template [tableRowTemplate]="transcripts()" let-row>
-    <td>{{ row.hearingDate | luxonDate: 'dd MMM yyyy' }}</td>
-    <td>{{ row.type }}</td>
-    <td>{{ row.requestedOn | luxonDate: 'dd MMM yyyy HH:mm:ss' }}</td>
-    <td>{{ row.requestedBy }}</td>
-    <td>
-      <app-govuk-tag [colour]="statusColours[row.status]">{{ row.status }}</app-govuk-tag>
-    </td>
-    <td>
-      @if (row.status === 'Complete') {
-        <a class="govuk-link" [routerLink]="['transcripts', row.id]">View</a>
-      }
-    </td>
+    @if (adminScreen()) {
+      <td>
+        <a
+          class="govuk-link"
+          [routerLink]="['/admin/transcripts', row.id]"
+          [queryParams]="{ backUrl: '/admin/case/' + caseId() }"
+          >{{ row.id }}</a
+        >
+      </td>
+      <td>{{ row.courtroom }}</td>
+      <td>{{ row.type }}</td>
+      <td>{{ row.requestedOn | luxonDate: 'dd MMM yyyy' }}</td>
```

Comment:
Need to display time as well (I know the ticket design is different, but we need it)